languages/translations/en.json

```diff
@@ -51,8 +51,8 @@
             }
         },
         "phrase": {
-            "two linked by distance": "{instruction_one} then in {distance} {instruction_two}",
-            "two linked": "{instruction_one} then {instruction_two}",
+            "two linked by distance": "{instruction_one}, then in {distance}, {instruction_two}",
```

Comment:
In my dream world, this would be: `"{instruction_one}. Then, in {distance}, {instruction_two}."` But maybe I'm being too pedantic?

Reply:
I hear you. I tend to omit the comma between “then” and “in” in more casual writing but include it in more formal writing. (Likewise with pauses in speech.) In this case, I suspect that the extra comma would make the synthesized speech a bit too choppy.

Reply:
I agree with you!